src/jdk.compiler/share/classes/com/sun/tools/javac/resources/launcher.properties

```
@@ -124,6 +124,12 @@ launcher.err.cant.access.main.method=\
 launcher.err.cant.find.constructor=\
     can''t find no argument constructor in class: {0}
```

This is a nitpick, but I noticed the double `'` in the message above. 

```suggestion
    can't find no argument constructor in class: {0}
```

---

That is the correct syntax, required by `MessageFormat`: [MessageFormat#quoting-rules-in-patterns-heading](https://docs.oracle.com/en/java/javase/25/docs/api/java.base/java/text/MessageFormat.html#quoting-rules-in-patterns-heading)